haproxy.py

```diff
@@ -18,44 +18,44 @@
 RECV_SIZE = 1024
 METRIC_TYPES = {
     'MaxConn': ('max_connections', 'gauge'),
-    'CumConns': ('connections', 'counter'),
-    'CumReq': ('requests', 'counter'),
+    'CumConns': ('connections', 'derive'),
```

Comment:
Hi Christoph
Could you explain why did you use derive types instead of counter?